### ISSUE TITLE
Add possibilities to evaluate problem quality

### ISSUE THREAD

**Issue by adbuerger** [adbuerger/PECas#13]
This means, how good is the problem suited for parameter estimation, when taking into account the orders of magnitude of the several parameters, the measurements, ... Can this be evaluated an presented to the user in an appropriate way?

**Comment by ghorn** [comment]
do you do compute the fisher information matrix?

**Comment by adbuerger** [comment]
@ghorn Yes, but I was thinking whether there could be something that can tell you about your chances of succeeding with your parameter estimation just from your setup, without having even something computed yet, like "your initial guess shows that you might get problems with scaling", or "if you don't measure that state, you won't be able to estimate this parameter" and something like that. Because I thinks for beginners - and not only for them - hints like that could be really useful.

**Comment by adbuerger** [comment]
@ghorn Did you at some time come across something that might can quantify such statements like the ones above, like, just guessing that, maybe one could use condition numbers in some way? Did you already see such approaches somewhere?

**Comment by ghorn** [comment]
I don't do anything that powerful. I can suggest scaling factors for the modeling step based on magnitudes of the solution variables.

One thing I do which has proved very useful is that I report which constraints are active in a human readable form. I don't think you really deal with constraints very much though, do you?

One thing I've wanted to do was to analyze LICQ and report when it is violated, but I have never tried it

**Comment by ghorn** [comment]
Another idea could be to suggest a discretization, both in # timesteps and collocation polynomial degree, in order to achieve a user-provided reltol/abstol.
